Appraisals

```diff
@@ -1,24 +1,6 @@
-# encoding: utf-8
-
 # Latest release of mainline Chef versions here.
-%w(10 12).each do |tv|
+%w[12 12.7 13 14].each do |tv|
```

Comment:
Should 12 be here in addition to 12.7?

Reply:
I mean, because your PR says `Chef < 12.7 compatibility has been removed`.

Reply:
Yes, the 12 entry actually targets `~> 12.0` which means latest 12: `12.22.5`

Reply:
👍